data/settings.yaml

```diff
@@ -2,5 +2,4 @@ settings:
     PREFER_DATES_FROM: 'current_period'
     SUPPORT_BEFORE_COMMON_ERA: False
     PREFER_DAY_OF_MONTH: 'current'
-    SKIP_TOKENS:
-        - t
\ No newline at end of file
+    SKIP_TOKENS: ["t"]
```

Comment:
Following `languages.yaml` styling, wherever we are dealing with characters which would be skipped we use *flow style sequences*. Those dealing with translation, use *block style sequences*. Made this change to be consistent with that styling.